src/pycolmap/estimators/cost_functions.cc

```diff
@@ -132,28 +137,51 @@ void BindCostFunctions(py::module& m_parent) {
         "Sampson error for two-view geometry.");
 
   m.def("AbsolutePosePriorCost",
-        &AbsolutePosePriorCostFunctor::Create<const Rigid3d&,
-                                              const Eigen::Matrix6d&>,
+        &AbsolutePosePriorCostFunctor::Create<const Rigid3d&>,
         "cam_from_world_prior"_a,
-        "cam_cov_from_world_prior"_a,
         "6-DoF error on the absolute camera pose.");
+  m.def("AbsolutePosePriorCost",
+        &CovarianceWeightedCostFunctor<AbsolutePosePriorCostFunctor>::Create<
+            const Rigid3d&>,
+        "cam_cov_from_world_prior"_a,
+        "cam_from_world_prior"_a,
+        "6-DoF error on the absolute camera pose with prior covariance.");
+
   m.def("AbsolutePosePositionPriorCost",
-        &AbsolutePosePositionPriorCostFunctor::Create<const Eigen::Vector3d&,
-                                                      const Eigen::Matrix3d&>,
+        &AbsolutePosePositionPriorCostFunctor::Create<const Eigen::Vector3d&>,
         "position_in_world_prior"_a,
-        "position_cov_in_world_prior"_a,
         "3-DoF error on the absolute camera pose's position.");
+  m.def(
+      "AbsolutePosePositionPriorCost",
+      &CovarianceWeightedCostFunctor<
+          AbsolutePosePositionPriorCostFunctor>::Create<const Eigen::Vector3d&>,
+      "position_cov_in_world_prior"_a,
```

Comment:
The ordering changes in the following interfaces are not ideal (in terms of not only compatibility but also that we cannot support identity covariance as default), but I guess we dont have a good way to get around for now.

Reply:
I think this is fine, because the standard cost function without the covariance argument is available. In this case, one should not use the CovarianceWeightedCostFunctor because it has some computational overhead.